src/index.js

```diff
@@ -56,7 +56,18 @@ module.exports = {
     'sort-vars': ['error'], // https://eslint.org/docs/rules/sort-vars
 
     // Unicorn
+    'unicorn/consistent-destructuring': 'off',
+    'unicorn/consistent-function-scoping': 'off',
+    'unicorn/no-array-callback-reference': 'off',
+    'unicorn/no-array-for-each': 'off',
+    'unicorn/no-array-reduce': 'off',
+    'unicorn/no-null': 'off',
+    'unicorn/no-useless-undefined': 'off',
+    'unicorn/numeric-separators-style': 'off',
```

Comment:
I think it help improve readability.

Reply:
If everyone's down for this, I'll do this one as well.

Personally, I think ik looks really weird. :-P

Reply:
I hate it with a passion. I vote against haha

Reply:
if you have numbers in your code that you need numeric separators I think you should reconsider your approach

Reply:
> if you have numbers in your code that you need numeric separators I think you should reconsider your approach

Exactly, and that's why it needs to stay on. And if you do have those large numbers, you'll need some kind of seperator style to make it more readable.

Reply:
so... @bramdeba @JaccoGoris 

2 to 1, don't enforce it?